src/core/observables/DPDStress.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef OBSERVABLES_DPDSTRESS_HPP
+#define OBSERVABLES_DPDSTRESS_HPP
+
+#include "Observable.hpp"
+#include "particle_data.hpp"
+#include "pressure.hpp"
+#include <vector>
+
+namespace Observables {
+
+class DPDStress : public Observable {
+public:
+  int n_values() const override { return 9; };
+  std::vector<double> operator()(PartCfg &partCfg) const override {
+    std::vector<double> res(n_values());
```

Comment:
Why is this needed?

Reply:
I actually looked at `observables/StressTensor.hpp` to get an example.